6-Typescript/server/tests/env/random.ts

```diff
@@ -0,0 +1,21 @@
+import { ITodo } from './../../interfaces/ITodo';
+import {v4 as uuidv4} from 'uuid'
+import chance from 'chance'
+
+export const createTodo = (): ITodo => {
+    return {
+    "todoText": `test${chance().integer()}`,
+    "todoId": createTodoId(),
+    "checked": false,
+    "userId": createTodoId()
+    }
+}
+export const createTodoWithMissingChecked = () => {
+    return {
+    "todoText": `test${chance().integer()}`,
+    "todoId": createTodoId(),
+    "userId": createTodoId()
+    }
+}
+export const createTodoId = ():string => uuidv4()
+export const createUserId = ():string => uuidv4()
```

Comment:
`generateId`?

Reply:
I understand that for now this is the same function, but if in the future maybe well change the way you generate userId or todoId so it will be easier to change just one of them.